### PR TITLE
Live to VOD example

### PR DESCRIPTION
**Pull request by gfronza**

### CODE REVIEW

File: examples/encoding/live/live_to_vod.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+TS_MUXING_1080P_ID = 'COPY AND PASTE'
+TS_MUXING_720P_ID = 'COPY AND PASTE'
+TS_MUXING_480P_ID = 'COPY AND PASTE'
+TS_MUXING_360P_ID = 'COPY AND PASTE'
+TS_MUXING_240P_ID = 'COPY AND PASTE'
+TS_MUXING_AUDIO_ID = 'COPY AND PASTE'
+
+# set the start segment number. If it is set to None the first segment is taken as start segment
+START_SEGMENT = None
```
Comment (msmole) [inline examples/encoding/live/live_to_vod.py]: There should also be a setting for END_SEGMENT?

Comment (gfronza) [reply examples/encoding/live/live_to_vod.py]: END_SEGMENT is not available in the bitmovin-python yet, but I can add and push to this same Pull Request.

Comment (gfronza) [reply examples/encoding/live/live_to_vod.py]: @msmole done.

**Comment by dominic-miglar** [comment]
please include that new end_segment_number property also in the unit tests for the corresponding resources

**Comment by gfronza** [comment]
@dominic-miglar done.

**Pull request merged**